packages/near-api-js/package.json

```diff
@@ -26,12 +26,12 @@
     "devDependencies": {
         "@types/bn.js": "^5.1.0",
         "@types/http-errors": "^1.6.1",
+        "@types/node": "^18.7.14",
```

Comment:
This was a [phantom dependency](https://rushjs.io/pages/advanced/phantom_deps/#phantom-dependencies) 🙀 lucky it was only types.